dub.json

```diff
@@ -21,7 +23,12 @@
 	"copyright": "Copyright (c) Ramon Santamaria (@raysan5), Petro Romanovych (@onroundit), Jan Hoenig (@m3m0ry), Steven Schveighoffer (@schveiguy), Liam McGillivray (@LiamM32)",
         "excludedSourceFiles": ["source/rlgl.d", "source/raymath.d", "source/easings.d", "source/raymathext.d", "source/raylib_types.d"],
 	"targetType": "sourceLibrary",
-        "subPackages" : [
-            "install"
-        ]
+    "subPackages": [
+        {
+            "name": "raygui",
+            "targetType": "library",
+            "dependencies": {"raylib-d": {"path": "."}},
+            "sourceFiles": ["source/raygui.d"]
```

Comment:
I don't think this is going to work. everything in `source` is already part of raylib-d. So this can't be a subpackage.

You also have removed the install subpackage, which isn't correct.

Can we remove this change for now? We can discuss whether raygui should be a separate package or not later. I think in fact it should be a *separate* package, as in, a separate git repository. Keeping it as part of the same repository ties it to the same release numbers.

Reply:
Sorry. It's been a long, long while. I have been away from home and busy with other things since.

I don't remember (if I ever knew) what the "install" package is.

I agree that Raygui would probably be better as a separate package, but until we do that, I figured that making it a subpackage would make sense, just for this one release.

I don't remember if it was something dependency-related. If you make this version of Raygui part of the same base package, does it still compile in Better-C mode?

Reply:
the install subpackage is the one that runs when you use `dub run raylib-d:install`